src/pretalx/common/templates/common/auth.html

```diff
@@ -28,11 +28,11 @@ <h4 class="text-center">{% translate "I already have an account" %}</h4>
                 {% translate "Log in" %}
             </button>
         {% endif %}
-        {% socialapp_exists 'eventyay' as eventyay_exists %}
+        {% socialapp_exists request.event.organiser.slug as eventyay_exists %}
```

Comment:
You expose `event` to context but why still use `request.event`?

Reply:
fixed.